### DISCUSSION TITLE
Dashboard-wide overrides

### DISCUSSION THREAD

**Discussion by echatman-ias** [grafana/grafana#71808] [Dashboards, Panels & Visualizations]
I often want to define overrides that apply to every panel in a dashboard (for example: all series named "foo" should be colored green). Copying them all over to every panel in the dashboard is very difficult and error prone. It would be fantastic if the dashboard configuration included an overrides section where I could define overrides that applied to all panels.

**Comment by echatman-ias** [comment]
Related to this issue, but I don't want global overrides - only dashboard-specific ones: https://github.com/grafana/grafana/issues/2245

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82627. Feel free to continue the discussion around this there. Thank you!
